src/main/java/redis/clients/jedis/UnifiedJedis.java

```diff
@@ -256,6 +256,10 @@ public final <T> T executeCommand(CommandObject<T> commandObject) {
     return executor.executeCommand(commandObject);
   }
 
+  public final <T> T executeCommandToReplica(CommandObject<T> commandObject) {
```

Comment:
Let's add minimal number of new methods.
I'm also thinking of adding these only in JedisCluster. When we can do replica read for single master + replicas mode as well, we can move these in UnifiedJedis.

Reply:
> Let's add minimal number of new methods.

I have deleted the other methods and have only kept this one `executeCommandToReplica(CommandObject)`. please review.

> I'm also thinking of adding these only in JedisCluster. When we can do replica read for single master + replicas mode as well, we can move these in UnifiedJedis.

Sorry, I don't understand what you mean. I think if we use JedisCluster to connect a master-replicas (not redis cluster) and will get an error `ERR This instance has cluster support disabled`, right?

Reply:
> > I'm also thinking of adding these only in JedisCluster. When we can do replica read for single master + replicas mode as well, we can move these in UnifiedJedis.
> 
> Sorry, I don't understand what you mean. I think if we use JedisCluster to connect a master-replicas (not redis cluster) and will get an error `ERR This instance has cluster support disabled`, right?

I meant moving the methods (well, at this moment, just this method) in JedisCluster instead of keeping in UnifiedJedis.

Reply:
I see. It's done